test/resources/mocks.ts

```diff
@@ -213,6 +213,33 @@ export function generateIdToken(overrides?: object, claims?: object): string {
   return jwt.sign(payload, certificateObject.private_key, options);
 }
 
+/**
+ * Generates a mocked Auth Blocking token.
+ *
+ * @param {object} overrides Overrides for the generated token's attributes.
+ * @param {object} claims Extra claims to add to the token.
+ * @return {string} A mocked Auth Blocking token with any provided overrides included.
```

Comment:
```suggestion
 * @returns  A mocked Auth Blocking token with any provided overrides included.
```
Types in curly braces are needed and are not supported by API Extractor. It will defer to the types expressed in the Typescript code.

Reply:
removed